app/src/main/java/org/wikipedia/readinglist/ReadingListsFragment.kt

```diff
@@ -60,12 +69,22 @@ class ReadingListsFragment : Fragment(), SortReadingListsDialog.Callback, Readin
     private val readingListItemCallback = ReadingListItemCallback()
     private val readingListPageItemCallback = ReadingListPageItemCallback()
     private val searchActionModeCallback = ReadingListsSearchCallback()
+    private val multiSelectModeCallback = MultiSelectCallback()
     private var actionMode: ActionMode? = null
     private val overflowCallback = OverflowCallback()
     private var currentSearchQuery: String? = null
     private var recentImportedReadingList: ReadingList? = null
+    private var selectMode: Boolean = false
+    private var importMode: Boolean = false
     private var shouldShowImportedSnackbar = false
 
+    val filePickerLauncher = registerForActivityResult(ActivityResultContracts.StartActivityForResult()) {
+            if (it.resultCode == AppCompatActivity.RESULT_OK) {
+                val uri: Uri = it.data?.data!!
```

Comment:
Type specification can be ignored and it would be safe to use `it.data?.data?.let {}` here.